dts/bindings/sensor/nxp,fxls8974-common.yaml

```diff
@@ -41,3 +41,14 @@ properties:
       - 8
       - 4
       - 2
+
+  expected-whoami-value:
```

Comment:
remove "expected" from the property name

Reply:
Updated.